.github/workflows/container_release.yaml

```diff
@@ -46,22 +46,18 @@ jobs:
         run: poetry install
 
       - name: Create output folder
-        run: mkdir -p results
+        run: mkdir -p data
 
       - name: List aws hourly images
         run: |
-          AWS_REGIONS=$(poetry run rhelocator-updater aws-regions)
-          regions=$(echo "$AWS_REGIONS" | jq -r '.[]')
-          for region in ${regions[@]}; do
-            poetry run rhelocator-updater aws-hourly-images --region $region >> ./results/aws_hourly_images.json
-          done
+          poetry run rhelocator-updater aws-hourly-images >> ./data/temp-aws-data.json
```

Comment:
That looks way less complicated :smiley:

Reply:
Yeah was about time, wasn't it? :D